scripts/test_all_fuzzers.sh

```diff
@@ -42,6 +42,11 @@ do
         continue
     fi
 
+    # Skip windows fuzzers
+    if [[ $fuzzer == *"windows"* ]]; then
+        continue
```

Comment:
I'd prefer it if the fuzzer "skips" itself in the Makefile.toml or something like that? IIRC we do the same for linux fuzzers on windows already

Reply:
The issue is that we need to skip the `check` because the windows fuzzer uses a feature which is not available on Linux:

```

Reply:
Can you hide the failing line behind a `#[cfg(windows)]`? That would be the best way